src/open_deep_research/tdd/run.py

```
@@ -0,0 +1,58 @@
+        
+    Returns:
+        The results of the TDD process
+    """
+    logger.info(f"Running TDD for query: {query}")
+    
+    # Create configuration if not provided
+    if config is None:
+        config = {}
+    
+    # Create TDD configuration
+    tdd_config = TDDConfiguration(**config)
```

Ruff (0.11.9)

Local variable `tdd_config` is assigned to but never used

Remove assignment to unused variable `tdd_config`

(F841)